portlets/src/main/webapp/vue-app/realizations/components/SelectPeriod.vue

```diff
@@ -149,10 +149,10 @@ export default {
       ];
     },
     fromDate() {
-      return this.value && new Date(this.value.min);
+      return this.value && new Date(this.value.min - (new Date().getTimezoneOffset() * 60000));
```

Comment:
Not really convinced by this kind of usage of Time zone. It's better to communicate with Server using RFC-3339 to allow the server to understand the date and used Time zone.

Reply:
@boubaker if am not mistaken, I think that this bug has been resolved by your commit https://github.com/Meeds-io/gamification/commit/0aaf994a84eeb50cfe57b11aaac7761704c61d25 , it's not reproduced in the ACC any more .

Reply:
Ah ok, so no need to this change anymore ? If yes, you can close this IMO.

Reply:
okay thank you .